pom.xml

```diff
@@ -170,24 +170,35 @@
          For example, if you imported Netty's BOM here, using Brave would also download that BOM as
          it depends indirectly on io.zipkin.zipkin2:zipkin. As Brave itself is indirectly used, this
          can be extremely confusing when people are troubleshooting library version assignments. -->
+    <dependencies>
+      <dependency>
+        <groupId>org.junit</groupId>
```

Comment:
Bringing JUnit BOM here since the runner requires `junit-platform-launcher` presence, which is versioned individually (f.e Jupiter 5.12.1, Launcher 1.12.1)

Reply:
I know it is annoying, but let's not import any boms in the top-level parent. there's a comment directly above explaining. it is annoying that there are paired versions, but it isn't the only pair we have, and this impacts anyone using the pom..